experiments/generic/disk_loss/disk_loss_k8s_job.yml

```diff
@@ -38,9 +38,10 @@ spec:
               fieldRef:
                 fieldPath: metadata.namespace
 
-          # Only GCP is supported
+          # GKE and AWS supported
```

Comment:
GKE and EKS supported

Reply:
Have you tested on EKS?

Reply:
You have tested, few days ago right?

Reply:
No, I have not tested disk-loss

Reply:
But I think you already tested it on aws. that's why I am adding aws

Reply:
Yes, Tested on AWS kubeADM.

Reply:
Make GKE and AWS KubeADM supported

Reply:
Here we are only mentioning about supporting cloud_platform. so I think ` GKE and AWS supported` is fine

Reply:
AWS also represent different types of K8s cluster like EKS, KubeADM and many more. Let's be specific

Reply:
we can mention details in docs, we have a separate section for that..

Reply:
oKAY FINE

Reply:
@rajdas98 Added AWS(kubeADM) here: https://github.com/litmuschaos/litmus-docs/pull/237